src/components/button.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+type BtnProps = {
+  text: string;
+  onClick: () => void;
+  idName?:string;
+};
+const Button:React.FC<BtnProps> = ({ idName, text, onClick }: BtnProps) => {
+  return (
+    <button className='btn' onClick={onClick} id={idName}>
+      {text}
+    </button>
+  );
+};
+
```

Comment:
For setting an id / classname for a component i'd do something like this: 

```suggestion

type BtnProps = {
  text: string;
  onClick: () => void;
};
const Button:React.FC<BtnProps> = ({ idName, text, onClick, ...restProps }) => {
  return (
    <button className='btn' onClick={onClick} {...restProps}>
      {text}
    </button>
  );
};

```